demo-projects/meetup/schema.js

```diff
@@ -20,7 +20,7 @@ const cloudinaryAdapter = new CloudinaryAdapter({
 });
 
 const access = {
-  userIsAdmin: ({ authentication: { item: user } }) => !!(user && user.isAdmin),
```

Comment:
Why not live dangerously and rely on falsy?

Reply:
Agreed, we don't seem to do this elsewhere..?

Reply:
I don't have a horse in this race, just prefer explicit `Boolean` over double bang.